nautobot/extras/plugins/urls.py

```diff
@@ -30,7 +30,7 @@
 for plugin_path in settings.PLUGINS:
     plugin_name = plugin_path.split(".")[-1]
     app = apps.get_app_config(plugin_name)
-    base_url = getattr(app, "base_url") or app.label
+    base_url = getattr(app, "base_url") or app.label.replace("_", "-")
```

Comment:
Given that (as stated in #337) `app.base_url` is always present (even if `None`) it seems like we shouldn't need to use `getattr` at all.

```suggestion
    base_url = app.base_url or app.label.replace("_", "-")
```

Same suggestion below.